DataSetExplorer/DataSetSerializer/IDataSetExporter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DataSetExplorer
+{
+    public interface IDataSetExporter
+    {
+        public void Export(string outputPath);
```

Comment:
Should return Result<string> so that we can have an error if, for example, the outputPath targets a folder for which the app doesn't have permission to access.